include/mbedtls/mbedtls_config.h

```diff
@@ -1364,6 +1364,15 @@
  */
 //#define MBEDTLS_PSA_CRYPTO_CLIENT
 
+/**
+ * \def MBEDTLS_PSA_ENABLE_KEY_ENROLLMENT
+ *
+ * Enable support for set/get key enrollment functions in PSA.
+ *
+ * \warning This is an Mbed TLS extension to the standard PSA interface.
+ */
+// #define MBEDTLS_PSA_ENABLE_KEY_ENROLLMENT
```

Comment:
For backward compatibility, the feature has to be available by default. So we need to make this a negative option, off by default.